src/hotspot/cpu/arm/frame_arm.inline.hpp

```diff
@@ -40,16 +40,17 @@ inline frame::frame() {
   _cb = NULL;
   _deopt_state = unknown;
   _on_heap = false;
+  _oop_map = NULL;
   DEBUG_ONLY(_frame_index = -1;)
 }
 
 inline frame::frame(intptr_t* sp) {
```

Comment:
Will AArch64 code work?

inline frame::frame(intptr_t* sp) : frame(sp, sp, *(intptr_t**)(sp - frame::sender_sp_offset), *(address*)(sp - 1)) {}

Reply:
Nice to have comments here to avoid spending time on it in the future, but it's not crucial.